_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -90,63 +84,47 @@ Single autocomplete term for select.
 
 #### Passed Example 3
 
-Autocomplete term, only valid for textarea.
+This `autocomplete` [attribute value][] only has the required token, and is valid for a `textarea` element. Missing upper and lower case letters is allwed for `autocomplete` attributes.
 
 ```html
 <textarea autocomplete="Street-Address"></textarea>
 ```
 
 #### Passed Example 4
 
-Two autocomplete terms.
+This `autocomplete` [attribute value][] has a `work` token, allowed because it is used before `email`. The `email` taken is allowed on `input` elements with a `type` [attribute value][] of `text`.
```

Comment:
```suggestion
This `autocomplete` [attribute value][] has a `work` token, allowed because it is used before `email`. The `email` token is allowed on `input` elements with a `type` [attribute value][] of `text`.
```